crates/ra_project_model/src/sysroot.rs

```diff
@@ -100,9 +99,15 @@ fn try_find_src_path(cargo_toml: &Path) -> Result<PathBuf> {
         .current_dir(cargo_toml.parent().unwrap())
         .args(&["--print", "sysroot"])
         .output()
-        .map_err(|e| format!("rustc --print sysroot failed: {}", e))?;
+        .map_err(|e| anyhow!("rustc --print sysroot failed: {}", e))?;
     if !rustc_output.status.success() {
-        Err("failed to locate sysroot")?;
+        let exit_message = match rustc_output.status.code() {
+            Some(code) => {
+                anyhow!("failed to locate sysroot: rustc --print sysroot exited with code {}", code)
```

Comment:
I *think* this can use `bail!` macro, rather than an `Err(...)?`  pattern.

Reply:
done